drivers/hmis_csv_twenty_twenty_four/app/models/hmis_csv_twenty_twenty_four/exporter/export_concern.rb

```diff
@@ -41,6 +41,23 @@ def self.replace_blank(row, hud_field:, default_value:)
       row
     end
 
+    def self.round_value(row, hud_field:, rounding:, positive:)
+      return row unless row[hud_field].present?
```

Comment:
should this `return row unless rounding.present?`

Reply:
I guess that's handled in `rounding_columns` so maybe not necessary.

Reply:
Yeah -- handled in `rounding_columns` -- but I safed it in the `case` in case it gets called directly.